src/Config/ConfigManager.cs

```diff
@@ -121,6 +122,10 @@ private static void CreateConfigElements()
                 "Should UnityEngine.Debug.Log messages be printed to UnityExplorer's log?",
                 false);
 
+            Log_To_Disk = new("Log To Disk",
+                "Should UnityExplorer save log files to the disk?",
+                false);
```

Comment:
May I ask that the default value be set to true, as it is undesirable to change the previous behavior?

Reply:
Yeah will do, I guessed that would be brought up tbf xD